pages/agent/v3/docker.md.erb

```diff
@@ -75,6 +75,28 @@ FROM buildkite/agent:3
 COPY hooks /buildkite/hooks/
 ```
 
+## Permissions Errors when using Docker
+
+A common problem encountered when using Docker for builds along with volume mounts from the host is the mismatch of permissions that can occur.
```

Comment:
What about?

> A problem you may encounter when using Docker volume mounts (`-v`), is that the container may create files on the host system with invalid user permissions, which can result in errors like the following:
>
>```bash
>$ git clean -fxdq
>warning: failed to remove dist/
>```

I idea is to try to get that error message searchable, because our agent does a `git clean` and that's usually when this problem crops up.